data/common.yaml

```diff
@@ -0,0 +1,3 @@
+---
```

Comment:
Are you able to put the migration to data in modules in another PR? There are quite a few changes in this PR that aren't related. Reviewing would be easier if this is in another PR.

Reply:
Potentially, although I didn't actually migrate any existing data to this structure. Currently only data related to the qdevice class is stored in hiera which is part of the Corosync-qdevice feature that is the focus of this PR.

That class is intended to be deployed on nodes which do not have the corosync cluster so I needed to configure it's defaults outside of the params class.

Reply:
I'm really not a fan of introducing data-in-modules for parts of the parameters. This change should be pulled out into a seperate PR and it should cover all parameters.

Reply:
No problem. I've unrolled it in the latest commit.